packages/vs-button/src/index.js

```diff
@@ -93,9 +95,10 @@ class VSButton extends Component {
       textTransform: 'uppercase',
       border: 'none',
       padding: `15px ${ this.sizeState() }`,
+      height: '48px',
       borderRadius: '4px',
       fontSize: '16px',
-      fontFamily: 'IntCircularProBook',
+      fontFamily: 'IntCircularPro, IntCircularProBook, Arial',
```

Comment:
Do we need both IntCircularPro and IntCircularProBook? You can probably get rid of book

Reply:
Book was what got imported for story book & the other is what Lumen uses. Aren't you guys using Book in ignite?

Reply:
I don't think it matters to ignite. I think it's more important to have them consistent.